packages/client/src/AccountsClient.js

```diff
@@ -73,11 +73,14 @@ export class AccountsClient {
     return this.getState().get('user').toJS();
   }
 
-  tokens(): TokensType {
-    return {
-      accessToken: this.storage.getItem(getTokenKey(ACCESS_TOKEN, this.options)),
-      refreshToken: this.storage.getItem(getTokenKey(REFRESH_TOKEN, this.options)),
-    };
+  async tokens(): Promise<TokensType> {
+    return Promise.all([
+      Promise.resolve(this.storage.getItem(getTokenKey(ACCESS_TOKEN, this.options))),
```

Comment:
I think that setItem and remove is async as well on AsyncStorage. Maybe write a wrapper class that hides any storage that is sync for all methods we use?